client/components/TestQueueCompletionStatusListItem/BotTestCompletionStatus/index.js

```diff
@@ -0,0 +1,52 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useTestPlanRunValidatedAssertionCounts } from '../../../hooks/useTestPlanRunValidatedAssertionCounts';
+
+const BotTestCompletionStatus = ({ testPlanRun, id, runnableTestsLength }) => {
+    const {
+        totalValidatedAssertions,
+        totalPossibleAssertions,
+        testResultsLength,
+        stopPolling
+    } = useTestPlanRunValidatedAssertionCounts(testPlanRun, 2000);
+
+    useEffect(() => {
+        if (testResultsLength === runnableTestsLength) {
+            stopPolling();
+        }
+    }, [testResultsLength, stopPolling]);
+
+    return (
+        <ul id={id} className="text-secondary">
+            <li>
+                {`Responses for ${testResultsLength} of ${runnableTestsLength} tests recorded`}
+            </li>
+            <li>
+                {`Verdicts for ${totalValidatedAssertions} of ${totalPossibleAssertions} assertions assigned`}
```

Comment:
Yeah, it only shows the assertions completed that could be completed based on which tests have actually returned results. What you suggest may feel more natural in some scenarios but It might be a little awkward for a canceled/running job where you don't want to imply that assertions could be verified for tests that didn't complete or haven't completed yet.

Reply:
Archived this feedback in [this comment](https://bocoup.slack.com/archives/CB9ESG2AF/p1703199005107479) for possible use on further design cycles.